src/libs/convertToLTR/index.ts

```diff
@@ -2,4 +2,18 @@ import ConvertToLTR from './types';
 
 const convertToLTR: ConvertToLTR = (text) => text;
 
+const moveCursorToEndOfLine = (
```

Comment:
Can you please add a comment to the top of this file that explains the difference between Android and the rest of the platforms? Something like this should be fine:

> The Android platform has to handle switching between LTR and RTL languages a bit differently (see the Android file for more details). For all other platforms, these can simply be no-op functions.

Reply:
I haven't found any official documentation of this, except for this one https://developer.android.com/training/basics/supporting-devices/languages, I have added the comment.